tests/e2e/contacts/new-lineage-structure.wdio-spec.js

```diff
@@ -34,22 +35,36 @@ describe('Create new lineage structure', () => {
   it('Create new health center', async () => {
     await contactPage.addPlace('district_hospital', centerName, centerContact);
     await sentinelUtils.waitForSentinel(); // prevent stale element references
-    expect(await contactPage.getPrimaryContactName()).toBe(centerContact);
+    chai.expect(await contactPage.getPrimaryContactName()).to.equal(centerContact);
   });
 
   it('Create new area', async () => {
     await contactPage.addPlace('health_center', area, areaContact);
     await sentinelUtils.waitForSentinel(); // prevent stale element references
-    expect(await contactPage.getPrimaryContactName()).toBe(areaContact);
+    chai.expect(await contactPage.getPrimaryContactName()).to.equal(areaContact);
   });
 
   it('Create new household', async () => {
     await contactPage.addPlace('clinic', household, householdContact);
     await sentinelUtils.waitForSentinel(); // prevent stale element references
-    expect(await contactPage.getPrimaryContactName()).toBe(householdContact);
+    chai.expect(await contactPage.getPrimaryContactName()).to.equal(householdContact);
```

Comment:
Why replace the baseline `expect()` with `chai.expect()`?  
I've tried running these tests locally with the baseline `expect()` and I can't see any difference :sweat_smile:

Reply:
I think that chai is more robust as an assertions library that the one that comes bundled with wdio.